docs/en/advanced/Scalability-Tuning.md

```diff
@@ -29,20 +29,17 @@ ALLUXIO_SECONDARY_MASTER_JAVA_OPTS+=" -Xms128g -Xmx256g "
 * As a rule of thumb set the min heap size to half the max heap size.
 * Each thread spawned by the master JVM requires off heap space determined by the thread stack size.
 When setting the heap size, ensure that there is enough memory allocated for off heap storage.
-For example, spawning `50000` threads with a default thread stack size of `1MB` requires at least
-`50 GB` of off-heap space available.
+For example, spawning `4000` threads with a default thread stack size of `1MB` requires at least
+`4 GB` of off-heap space available.
 
 ### Thread Pool Size
 
-An executor pool is used on the master to handle concurrent client requests. If you expect a large
-number of concurrent clients communicating with the master, tune the thread pool size by modifying
-the following properties. The actual thread count depends on the maximum concurrency expected on the
-cluster. For example, if spawning `10000` concurrent tasks (clients) each with a client thread
-pool size of `4`, set the master thread pool max to greater than `40000`.
+An executor pool is used on the master to handle concurrent client requests. You can tune the thread pool size
+by modifying the following property. The actual thread count depends on number of cores available on the master.
+For example, if the master has 64 cores, set the master thread pool max to greater than `512`.
```

Comment:
Can we say "Executor thread pool instead" and then put the second sentence last so it read 

"An executor thread pool is used... . The actual thread count ... For example, .... You can tune the thread pool size by modifying the following property"

Reply:
Makes sense, updated.